xbmc/cores/paplayer/VideoPlayerCodec.h

```diff
@@ -41,24 +41,24 @@ class VideoPlayerCodec : public ICodec
 private:
   CAEStreamInfo::DataType GetPassthroughStreamType(AVCodecID codecId, int samplerate, int profile);
 
-  CDVDDemux* m_pDemuxer;
+  CDVDDemux* m_pDemuxer{};
   std::shared_ptr<CDVDInputStream> m_pInputStream;
   std::unique_ptr<CDVDAudioCodec> m_pAudioCodec;
 
   std::string m_strContentType;
   std::string m_strFileName;
-  int m_nAudioStream;
-  size_t m_nDecodedLen;
+  int m_nAudioStream{-1};
+  size_t m_nDecodedLen{};
 
-  bool m_bInited;
-  bool m_bCanSeek;
+  bool m_bInited{};
+  bool m_bCanSeek{};
 
-  ActiveAE::IAEResample *m_pResampler;
-  DVDAudioFrame m_audioFrame;
-  int m_planes;
-  bool m_needConvert;
-  AEAudioFormat m_srcFormat;
-  int m_channels;
+  ActiveAE::IAEResample* m_pResampler{};
+  DVDAudioFrame m_audioFrame{};
+  int m_planes{};
+  bool m_needConvert{};
```

Comment:
next style nit/hint, i‘d prefer to be explicit with all those e.g. `{false}`

Reply:
I would agree but it looks like the code guidelines prefer the empty braces (https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#54-initialization)? But then in a later example it's explicit again (https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#114-default-member-initialization)? :thinking:

Reply:
uh yeah, please decide at your leisure :)

Reply:
imo i suggest to improve our docs to use only explicits init values
are more readable when you investigate for something

Reply:
I agree, I think that explicit values are better for when it's a primitive type, empty elsewhere, and avoided when possible, e.g. don't do `std::string example{}` because std::string has a default constructor. Doc improvement would be good.